_layouts/downloads.html

```diff
@@ -472,11 +472,6 @@ <h5 class="section-toggle section-toggle-start-open ">Provider Packages</h5>
                         </div>
                       </div>
 
-                      <!--
```

Comment:
Please put back in the block commenting out the cloudant incubating release. These links are dead (the download was removed from the dist server and now is only in the archive).  Removing the release was I think an inadvertent action by someone, but having the dead link on the downloads page is not useful (and got flagged #464 ).